package.json

```
@@ -5,8 +5,8 @@
     "repository": "https://github.com/trezor/trezor-suite.git",
     "license": "SEE LICENSE IN LICENSE.md",
     "engines": {
-        "node": ">=14",
-        "yarn": ">=1.22.0"
+        "node": "~14.18.1",
+        "yarn": "~1.22.17"
```

If we are pinning to particular versions shouldn't this just be the following?

```suggestion
        "node": "14.18.1",
        "yarn": "1.22.17"
```

---

I think that patch version doesn't matter that much, but we can pin exact version.